app/assets/javascripts/templates/bulk_buy_modal.html.haml

```diff
@@ -0,0 +1,34 @@
+.row
+  .columns.small-12
+    %h3{"ng-bind" => "::variant.extended_name"}
+
+.row.variant-bulk-buy-price-summary
+  .columns.small-6
+    .variant-unit {{ ::variant.unit_to_display }}
+  .columns.small-6
+    {{ variant.line_item.total_price | localizeCurrency }}
+
+.row
+  .columns.small-6
+    .variant-bulk-buy-quantity-label
+      Min quantity
```

Comment:
this needs to be translated right?

Reply:
Right! I feel like such a n00b! That happens when commits become too big and you rush things...

Reply:
yeah, even when the PR is big, it's useful to keep the commits small. but the natural tendency is to group things together :+1: